exchanges/bybit/bybit_websocket.go

```diff
@@ -65,7 +65,6 @@ var defaultSubscriptions = subscription.List{
 	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyOrdersChannel},
 	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyWalletChannel},
 	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyTradesChannel},
-	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: chanPositions},
```

Comment:
NOTE: This is associated with linear/inverse/options and can be removed

Reply:
That was going to be my first question 😄